src/interactions/User/AsEndUser.js

```diff
@@ -51,6 +54,28 @@ export class AsEndUser {
     return this.isLoggedIn() &&
            !!_find(this.user.notifications, {isRead: false})
   }
+
+  /**
+   * Returns a mr-text-* color for the user's displayName
+   */
+  colorCode() {
+    return mapColors(this.user.osmProfile.displayName)
```

Comment:
While this is probably fine, you might consider a lodash `_get` here just to be safe